pkg/proxy/replica.go

```diff
@@ -78,7 +78,25 @@ func (p *Proxy) ReplicaRebuildingStatus(ctx context.Context, req *rpc.ProxyEngin
 		Status: make(map[string]*eptypes.ReplicaRebuildStatusResponse),
 	}
```

Comment:
NIT: same here. don't need to init the response at the beginning.

Reply:
Updated.